fy22_ferry_study/attraction_travel_times/distance_direction_parser.py

```diff
@@ -0,0 +1,164 @@
+import googlemaps
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+import pandas as pd
+from shapely.geometry import MultiLineString
+import polyline
+import geopandas as gpd
+
+load_dotenv()
+api_key = os.getenv("google_api")
+gmaps = googlemaps.Client(key=api_key)
+GDRIVE_FOLDER = Path(os.getenv("GDRIVE_FOLDER"))
+origins = GDRIVE_FOLDER / "origin.csv"
+destinations = GDRIVE_FOLDER / "attractions.csv"
+origin_df = pd.read_csv(origins)
+destinations_df = pd.read_csv(destinations)
+now = datetime.now()
+
+
+origin = (
+    39.946196,
+    -75.139832,
+)  # hardcoded here since my origin doesn't change, but if yours does you can tweak the function to accept origins.
+
+
+def distance_duration_iteration(mode):
+    """returns a list of origin/destination dicts for driving or transit, respectively"""
+    driving_list = []
+    transit_list = []
+
+    def distance_duration(destination):
+        output = gmaps.directions(
+            origin,
+            destination,
+            mode,
+            units="imperial",
+            departure_time=now,
+        )
+        names = {"name": name}
+        output.insert(0, names)
+
+        if mode == "driving":
+            driving_list.append(output)
+
+        if mode == "transit":
+            transit_list.append(output)
+
+    """loops through the destinations file and runs the distance duration function"""
```

Comment:
Docstrings should only show up once per function, right at the top. If you need additional comments within use `# comment` instead of `""" comment """`